apps/src/applab/applab.js

```diff
@@ -884,6 +890,10 @@ function setupReduxSubscribers(store) {
     );
```

Comment:
Not blocking: This function is starting to get pretty lengthly. Might be worth a quick refactor.

Reply:
There's a lot going on with how we interact with redux from applab.js. I agree it's worth refactoring, but I think it will not be quick, and we should consider/plan it from a higher level.